Servers/set-server-disks.md

```diff
@@ -57,17 +57,17 @@ Use this API operation when you want to change the disks on an existing server.
           "value":[
              {
                "diskId":"0:0",
-               "sizeGB":1
+               "sizeGB":"1"
 			   "type":"raw"
              },
              {
                "diskId":"0:1",
-               "sizeGB":2
+               "sizeGB":"2"
 			   "type":"raw"
              },
              {
                "diskId":"0:2",
-               "sizeGB":16
+               "sizeGB":"16"
```

Comment:
Add a comma at end to make it valid JSON

Reply:
,